shared/logutil/logutil.go

```diff
@@ -23,3 +26,21 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+// CountdownToGenesis logs the time remains until genesis time
+func CountdownToGenesis(genesisTime time.Time, secondsCount int) {
+	ticker := time.NewTicker(time.Duration(secondsCount) * time.Second)
+
+	for {
+		select {
+		case <-time.NewTimer(genesisTime.Sub(roughtime.Now()) + 1).C:
+			// Exit upon hitting the genesis time.
+			log.Infof("genesis time\n")
+			return
+
+		case <-ticker.C:
+			// Exit if we have already reached the genesis time, else log:
```

Comment:
This comment is confusing and doesn't add additional informaiton. Please consider removing it or clarifying the comment.

Reply:
Okay will remove it, better